### ISSUE TITLE
Notify on new failed tests?

### ISSUE THREAD

**Issue by jeanconn** [sot/skare3_tools#77]
Should the dashboard test code notify on new failed tests?  Or should that just be plugged in to the old-school jobwatch/skawatch (could be set to read the test.log and look for "something")?

**Comment by taldcroft** [comment]
Yes, we (aca@cfa) should get an email for any failed tests.  I don't see an immediate need to limit notifications to only *new* failed tests.

**Comment by jeanconn** [comment]
See I was optimistic for once and figured any failed tests would have to be new!

**Comment by taldcroft** [comment]
I was assuming you were considering the case of a failed test that doesn't get fixed for a month.

**Comment by jeanconn** [comment]
If we end up there, I'd be comfortable pushing a test change to master that stops the test until it can be repaired more thoroughly... but we might disagree on philosophy at that point.
